sdks/python/apache_beam/examples/streaming_wordcount.py

```diff
@@ -25,16 +25,19 @@
 
 import argparse
 import logging
-import re
 
 
 import apache_beam as beam
 import apache_beam.transforms.window as window
 
 
+def split_fn(lines):
```

Comment:
The changes in this file are not reflected in the description of the pull request.